tekton/tasks/build-source-container.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: build-source-container
+spec:
+  description: >-
+    OSBS task for building source container image
+  params:
+    - name: OSBS_IMAGE
+      description: The location of the OSBS builder image (FQDN pullspec)
+      type: string
+    - name: USER_PARAMS
+      type: string
+      description: User parameters in JSON format
+
+  workspaces:
+    - name: ws-build-dir
+    - name: ws-context-dir
+    - name: ws-registries-secret  # access with $(workspaces.ws-registries-secret.path)/token
+    - name: ws-koji-secret  # access with $(workspaces.ws-koji-secret.path)/token
+    - name: ws-reactor-config-map
+
+  steps:
+    - name: build-source-container-image
+      image: $(params.OSBS_IMAGE)
+      script: >
+        atomic-reactor task source-build
+        --user-params="$(params.USER_PARAMS)"
+        --build-dir=$(workspaces.ws-build-dir.path)
+        --context-dir=$(workspaces.ws-context-dir.path)
+        --config-file=$(workspaces.ws-reactor-config-map.path)/config.yaml
```

In the current implementation, only the `task` subparser accepts the common arguments
```suggestion
        atomic-reactor task
        --user-params="$(params.USER_PARAMS)"
        --build-dir=$(workspaces.ws-build-dir.path)
        --context-dir=$(workspaces.ws-context-dir.path)
        --config-file=$(workspaces.ws-reactor-config-map.path)/config.yaml
         source-build
```